1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,7 +21,7 @@ let basketballTeam = {
 */
 
 // write code here
-
+console.log(basketballTeam.topPlayers)
```

Comment:
here you are expected to sort through this list of `basketballTeam.topPlayers` and then print each of the players on a new line. You can do that with array methods `.sort()` and `.forEach()`